docs/src/apalache/features.md

```diff
@@ -62,7 +62,7 @@ Operator  | Supported? | Milestone | Comment
 `f[e]` | ✔ | - |
 `DOMAIN f` | ✔ | - |
 `[ x \in S ↦ e]` | ✔ | - |
-`[ S -> T ]` | ✔ | - | Sometimes, the functions sets are expanded
+`[ S -> T ]` | ✔ | - | Sometimes, the functions sets are expanded <!-- What does this mean? -->
```

Comment:
I don't know what this comment is supposed to mean or what the impact is on users.

Reply:
It is confusing. It means that in some cases functions sets are treated purely symbolically, but sometimes the model checker would try to construct an explicit set of functions, which is exponential. Actually, the model checker would just fail there and will not try to construct such a set. So this sentence is not helping the user.

Reply:
Cool. That helps. How about

> Supported, provided the function can be interpreted symbolically